python_modules/dagstermill/dagstermill/dagstermill_tests/CLI_commands.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Let's invoke these from pytest tests and ensure that the right files are created and so forth.

A couple different ways to structure this. One is to make the click-decorated functions trivial wrappers around other functions that are invoked in test. See dagster/cli/cli_tests/test_cli_commands.py for examples this.

Upside of this is that it doesn't launch or fork its own process and tests all business logic. Downside is that it doesn't test the click decorators themselves. (We should probably add these tests in dagster/dagit tbh)

You can also launch new processes via the python `subprocess` module. See check_cli_execute_file_pipeline for this. In that case I just checked the exist code by using subprocess.check_output. However here we would also want to check the contents of the scaffolded content.